convert2rhel/actions/__init__.py

```diff
@@ -0,0 +1,619 @@
+# Copyright(C) 2016 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+
+import abc
+import collections
+import importlib
+import itertools
+import logging
+import os
+import os.path
+import pkgutil
+import re
+import traceback
+
+from functools import wraps
+
+import six
+
+from convert2rhel import utils
+from convert2rhel.utils import format_sequence_as_message
+
+
+logger = logging.getLogger(__name__)
+
+KERNEL_REPO_RE = re.compile(r"^.+:(?P<version>.+).el.+$")
+KERNEL_REPO_VER_SPLIT_RE = re.compile(r"\W+")
+BAD_KERNEL_RELEASE_SUBSTRINGS = ("uek", "rt", "linode")
+
+RPM_GPG_KEY_PATH = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
+# The SSL certificate of the https://cdn.redhat.com/ server
+SSL_CERT_PATH = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
+
+
+LINK_KMODS_RH_POLICY = "https://access.redhat.com/third-party-software-support"
+LINK_PREVENT_KMODS_FROM_LOADING = "https://access.redhat.com/solutions/41278"
```

Comment:
Those constants are not being used anywhere here. It's legacy code from checks.py